patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -560,7 +560,28 @@
              {
                  return false;
              }
-@@ -2011,6 +2167,11 @@
+@@ -1971,7 +2127,19 @@
+                 for (int j4 = j3; j4 < k3; ++j4)
+                 {
+                     IBlockState iblockstate1 = this.func_180495_p(blockpos$pooledmutableblockpos.func_181079_c(l3, i4, j4));
+-
++                    
```

Comment:
The formatter is somewhat funky and it removed a line... Be careful next time, this is okay

Reply:
This newline has spaces. Can you fix that?

Reply:
?